src/snapshots/pass/underscore_in_regular_annotations.md

```diff
@@ -245,6 +246,16 @@ process = |list| "processed"
            ^^^^
 
 
+**TOO MANY ARGS**
+The type _Result_expectts 0 argument, but got 2 instead.
```

Comment:
Typo in `expectts` and missing space before it.

Reply:
thanks! fixed